File: dsa/array/2pointer/Two_sum.java
public class Two_sum{
    public static void main(String[] args) {
        int []arr={0,2,3,5,7};
        int left=0;
        int right=arr.length-1;
        int target=5;
        while(left<right){
            int sum=arr[left]+arr[right];
            if(sum==target){
                System.out.println("[" + arr[left] + ", " +arr[right] + "]");
                left ++;
                right--;
            }else if(sum>target){
                right--;
            }else{
                left ++;
            }
        }
    }
}

File: dsa/array/2pointer/two_sum.py
arr=[1,3,5,8,9]
target=10
left=0
right=len(arr)-1

while left < right:
    sum1 = arr[left] + arr[right]
    if sum1 == target:
        print([left+1, right+1],end=" ")
        left+=1
        right-=1
    elif sum1 > target:
        right -= 1 
    else:
        left += 1   


